playbook/app/helpers/playbook/pb_sample_helper.rb

```diff
@@ -28,6 +28,15 @@ def get_sample_code_content(sample, type)
       raw rouge(code, rouge_type)
     end
 
+    def get_category(sample)
+      sample_yaml = YAML.load_file("#{Playbook::Engine.root}/app/pb_kits/playbook/data/samples.yml")
```

Comment:
On these paths, I'm wondering if we couldn't see about making them configuration vars then loading them that way vs. hard-coding them.

Reply:
Good idea, could be done with the menu yaml as well.